lib/emis/military_information_configuration_v2.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/soap'
+
+module EMIS
+  # Configuration for {EMIS::MilitaryInformationService}
+  # includes API URL and breakers service name.
+  #
+  class MilitaryInformationConfigurationV2 < Configuration
+    # Military Information Service URL
+    # @return [String] Military Information Service URL
+    def base_path
+      URI.join(Settings.emis.host, Settings.emis.military_information_url.v2).to_s
+    end
+
+    # :nocov:
+
+    # Military Information Service breakers name
+    # @return [String] Military Information Service breakers name
+    def service_name
+      'EmisMilitaryInformation'
```

Comment:
Do you think this this service name should be versioned? I'm still unclear on how exactly these service names are used in Breakers.